stats/kernel-selftests

```diff
@@ -104,6 +104,30 @@ end
 
 class NetStater < Stater
   def stat(line, stats)
+    if @test_script == 'udpgro_fwd.sh'
+      if @test_case && !@test_subcase && line !~ /perf/ && line =~ /GRO/ && line =~ /^# (.*)/
```

Comment:
here consider to combine the regexp of "line =~ /GRO/ && line =~ /^# (.*)/" to be single one